app/workers/talk_admin_create_worker.rb

```diff
@@ -3,6 +3,8 @@ class TalkAdminCreateWorker
 
   def perform(project_id)
     Project.find(project_id).create_talk_admin(client)
+  rescue TalkApiClient::NoTalkHostError => e
+    Rails.logger.info e
```

Comment:
I can, but that's just going to clutter up honeybadger with notifications. We can stop rescuing this error when all the Talk environments are deployed and it'll go through to Honeybadger normally.